flow/include/flow/Arena.h

```diff
@@ -157,23 +162,27 @@ struct ArenaBlockRef {
 	uint32_t nextBlockOffset;
 };
 
+FDB_DECLARE_BOOLEAN_PARAM(IsSecureMem);
+
 struct ArenaBlock : NonCopyable, ThreadSafeReferenceCounted<ArenaBlock> {
 	enum {
 		SMALL = 64,
 		LARGE = 8193 // If size == used == LARGE, then use hugeSize, hugeUsed
 	};
 
-	enum { NOT_TINY = 255, TINY_HEADER = 6 };
+	enum { NOT_TINY = 127, TINY_HEADER = 6 };
```

Comment:
Does this change affect performance characteristics significantly?

Reply:
No, `SMALL` allocations are still capped at 64B, so it can still be expressed with max value of `127`.
Added cost is an extra instruction to translate `ArenaBlock::tinySize` from 7-bit value to a 8-bit value.